packages/block-library/src/navigation-submenu/edit.js

```diff
@@ -422,17 +422,23 @@ export default function NavigationSubmenuEdit( {
 						onChange={ ( titleValue ) => {
 							setAttributes( { title: titleValue } );
 						} }
-						label={ __( 'Link title' ) }
+						label={ __( 'Title attribute' ) }
 						autoComplete="off"
+						help={ __(
+							'Provide additional information to help clarify or further describe the purpose of the link.'
+						) }
 					/>
 					<TextControl
 						__nextHasNoMarginBottom
 						value={ rel || '' }
 						onChange={ ( relValue ) => {
 							setAttributes( { rel: relValue } );
 						} }
-						label={ __( 'Link rel' ) }
+						label={ __( 'Rel attribute' ) }
 						autoComplete="off"
+						help={ __(
+							'The relationship of the linked URL as space-separated link types.'
```

Comment:
```suggestion
							'The relationship between a the link and the current post.'
```

This description made my think it might not be clear enough. 

This is a _suggestion_ based on https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/rel.

Still open for debate.

Reply:
Thanks Dave, the problem I see with referencing `post` is that the link can be to any content (post, page, product, and even a custom link).

Reply:
Is it the relationship "of the content to the link" or "of the link to the content"? What should rel be? Also what is "space-separated link types"?

Reply:
From [the article](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/rel) referenced by Dave: "The rel attribute defines the relationship between a linked resource and the current document.". It's an indication of how the linked document should be related or interpreted in the current page's context.

For example, `author` would be related to the author of the current document or article. `nofollow` is a recommendation from the current page/content that the link shouldn't be followed, by not endorsing it.

Space-separated link types would be the ones listed there: `author`, `nofollow`, `canonical`, etc.

Reply:
Yes the referenced article is the source of knowledge.

> the relationship between a linked resource and the current document.

Reply:
I don't know that most people will understand "space-separated link types" but I'm fine to iterate on this later.